Source/Core/DiscIO/WbfsBlob.h

```diff
@@ -19,7 +19,10 @@ class WbfsFileReader : public IBlobReader
 public:
 	static WbfsFileReader* Create(const std::string& filename);
 
-	u64 GetDataSize() const override { return m_size; }
+	// The WBFS format does not save the original file size.
+	// This function returns a constant upper bound
+	// (the size of a double-layer Wii disc).
```

Comment:
Well, a hash of a WBFS is potentially useful, as all WBFS files generated by good WBFS implementation of the disk with a good hash will produce the same hash. Potentially a database of good WBFS internal hashes could be created.

In which case assuming all WBFSs are padded two a 2 layer disk is not the worst assumptio, but if we make such an assumption we are stuck with it.